lib/public/views/Logs/Create/LogCreationModel.js

```diff
@@ -32,6 +32,8 @@ export class LogCreationModel extends Observable {
 
         this._onCreation = onCreation;
 
+        this._logType = 'New log';
+        this._titleLabelText = 'Title*';
```

Comment:
I think these two information do not belongs to the model, because they can be fully deduced from `this._parentLog`. They should be computed directly in the view without using intermediate variable.